ui/src/services/NodeVolumesUtils.js

```diff
@@ -211,23 +211,23 @@ export const getVolumeListData = createSelector(
           (alert) =>
             alert.labels.persistentvolumeclaim === volumePVC.metadata.name,
         );
-      } else {
-        volumeHealth = STATUS_NONE;
-      }
 
-      // THE RULES TO COMPUTE THE HEALTH
-      // compute the volume health based on the severity of the alerts
-      // critical => if there is at least one critical
-      if (volumeAlerts.length) {
-        volumeHealth = volumeAlerts.find(
-          (vol) => vol.labels.severity === STATUS_CRITICAL,
-        )
-          ? STATUS_CRITICAL
-          : STATUS_WARNING;
-      } else if (volumeComputedCondition === ('exclamation' || 'unlink')) {
-        volumeHealth = STATUS_NONE;
+        // THE RULES TO COMPUTE THE HEALTH
+        // compute the volume health based on the severity of the alerts
+        // critical => if there is at least one critical
+        if (volumeAlerts.length) {
+          volumeHealth = volumeAlerts.find(
```

Comment:
nitpick: instead of using `find`, I would rather use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).

> The some() method tests whether at least one element in the array passes the test implemented by the provided function. It returns a Boolean value. 

Reason being: you don't want to **find** the first critical alert (i.e you don't care about the alert object itself), you only want to know if there are **some** critical alerts in the array.
=> it's more logical to use `some` than `find`.

Reply:
Thanks Sylvain!! It definitely makes sense, we just need a Boolean to tell us if there is a critical alert.